po/it.po

```diff
@@ -266,23 +271,23 @@ msgstr "_Salva"
 
 #: pdfarranger/pdfarranger.py:934
 msgid "Discard changes and close?"
-msgstr ""
+msgstr "Scarto le modifiche e chiudo?"
 
 #: pdfarranger/pdfarranger.py:935 data/menu.ui.h:45
 msgid "_Close"
-msgstr ""
+msgstr "_Chiudi"
 
 #: pdfarranger/pdfarranger.py:940
 msgid "Save changes to “{}” before closing?"
-msgstr "Salvare le modifiche apportate a “{}”prima di chiudere?"
+msgstr "Salvo le modifiche apportate a “{}”prima di chiudere?"
 
 #: pdfarranger/pdfarranger.py:943
 msgid "Save changes before closing?"
-msgstr "Salvare le modifiche prima di chiudere?"
+msgstr "Salvo le modifiche prima di chiudere?"
```

Comment:
It looks like you are changing the grammar here. Is this consistent with how other software, ideally Gnome software, is using it?

Reply:
I did a search and saw that many translations use the infinitive form, I think gnome too. Do I have to change them all?

Reply:
I don't know about Italian but most software uses the infinitive form. If the remains off my Latin are of any help, you are changing this away from the infinitive form to some other form. Why do you think this is better? I think we should go with what most software does unless there are good reasons or a (Gnome) guideline saying otherwise.

Maybe the other translators, @MichelePezza or @albanobattistella, can chime in with there opinions?

Reply:
HI,

"Salvare le modifiche prima di chiudere?"  thisi is the correct form.

Reply:
It's "Inverti ordine" without uppercase "O", the same as "Seleziona tutto" and not "Selezionare tutto".
I finish now to revert those changes. 
How can I do to update file without create a new pull request?